cloud-chatops/tests/test_read_data.py

```diff
@@ -99,5 +111,15 @@ def test_validate_required_files_fail_users(mock_get_token, mock_get_config):
     """Test the validate files function"""
     mock_get_token.side_effect = ["mock_bot", "mock_app", "mock_github"]
     mock_get_config.side_effect = [{"owner1": ["repo1"]}, {}]
-    with pytest.raises(UserMapNotGiven):
+    with pytest.raises(ErrorInConfig):
+        validate_required_files()
+
+
+@patch("read_data.get_config")
+@patch("read_data.get_token")
+def test_validate_required_files_fail_channel(mock_get_token, mock_get_config):
+    """Test the validate files function"""
+    mock_get_token.side_effect = ["mock_bot", "mock_app", "mock_github"]
```

Comment:
I think the warning from codecov is valid, I can't see us access `channel` in any tests to check an error is thrown?

Reply:
Same as Megans comment. The test is there but not set up right. I have fixed this